### PR TITLE
Removing the sidebar and making custom graphs at tab at the top for the

### PR DESCRIPTION
**Pull request by branden14k**
Removing the sidebar and making custom graphs at tab at the top for the time being. Data still not loading correctly from the api though. Fyi The branch name is remove chart js but I did not do that and I forgot to create a branch with an accurate description. Sorry

### CODE REVIEW

**Comment by BenTheElder** [comment]
This looks pretty good to me, nothing that this switches to the non-cached URL, which we may need to switch back from in the future. I will notify everyone when the caching memory issue is resolved.

File: static/dashboard.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ $(document).ready(function () {
         }]
     };
 
-    $.getJSON('/api/cached/data/filtered/dogs', function (data) {
+    $.getJSON('/api/data/filtered/dogs', function (data) {
```
Comment (BenTheElder) [inline static/dashboard.js]: please revert this to the cached api, this is now fixed by #54

File: static/index.html [hunk 2]
```diff
@@ -286,7 +286,7 @@ <h4 align="center">Rest VS Active</h4>
       var mybarChart2;
       $.ajax({
                 type: "GET",
-                url: "/api/cached/data/filtered/dogs",
+                url: "/api/data/filtered/dogs",
```
Comment (BenTheElder) [inline static/index.html]: same here.

**Comment by BenTheElder** [comment]
I will merge this once it is switched back to the /cached/ apis. 👍

**Comment by BenTheElder** [comment]
ping @william7hsu, please update these.

**Comment by branden14k** [comment]
return the cached urls for pulling the data

**Comment by BenTheElder** [comment]
LGTM, merging.

**Pull request merged**